rest-server/engines/src/main/scala/com/actionml/engines/ur/URAlgorithm.scala

```diff
@@ -71,6 +89,18 @@ class URAlgorithm(dataset: URDataset)
 
 }
 
+object URAlgorithm {
+
+  case class URAlgorithmParams(comment: String,
+                               esMaster: String,
+                               indexName: String,
+                               typeName: String,
+                               availableDateName: String,
+                               expireDateName: String,
+                               dateName: String,
+                               num: String)
+}
+
```

Comment:
wrong style, see https://docs.scala-lang.org/style/indentation.html#methods-with-numerous-arguments for methods and https://docs.scala-lang.org/style/declarations.html for class declarations